### DISCUSSION TITLE
process log is not showing in action

### DISCUSSION THREAD

**Discussion by sparrow-code** [community/community#31208] [Actions]
process log is not showing in action tab on mine first account 

and working fine on mine another account

and i have github premium on mine first account still it is not showing

![process and Logs](https://user-images.githubusercontent.com/64479884/187153600-5fa00782-2907-4b7c-8183-4552a4e1a32d.PNG)

**Comment by Simran-B** [comment]
Does the log information show up once the workflow has finished?

**Reply by sparrow-code** [reply]
Yes Finish With Error Stopped 

And Still Showing Nothing
